src/main/java/com/google/sps/servlets/DeleteRecipeServlet.java

```diff
@@ -0,0 +1,26 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Entity;
+import java.io.IOException;
+import java.util.*;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet responsible for deleting recipes. */
+@WebServlet("/delete-recipe")
+public class DeleteRecipeServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    long id = Long.parseLong(request.getParameter("id"));
+
+    Key taskEntityKey = KeyFactory.createKey("PrivateRecipe", id);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    datastore.delete(taskEntityKey);
```

Comment:
Does datastore handle delete gracefully? ie what happens if it can't find the key to delete?

Reply:
I don't know and the documentation doesn't explicitly say
stackoverflow says that you can make a key for an entity even if the entity doesn't exist, so if I do so then delete the key I think it should be fine?

Reply:
Looks like it might throw an IllegalArgumentException: https://cloud.google.com/appengine/docs/standard/java/javadoc/com/google/appengine/api/datastore/DatastoreService.html#delete

You'll probably want to check that the key exists first before trying to delete it (I know you already merged this so feel free to do this in your next PR)